todo_app/test_view_model.py

```diff
@@ -0,0 +1,72 @@
+import pytest
```

Comment:
It's usually worth having your tests in a separate folder so you don't clutter the rest of your code